index.html

```diff
@@ -79,6 +79,37 @@ <h4 class="bd-logo-title">Validatore specifiche OpenAPI (beta 0.1)</h4>
                     <div class="it-socials d-none d-md-flex">
                       <!-- <span class="d-none d-lg-inline">Seguici su</span> -->
                       <ul>
+
+                        <li class="list-inline-item">
```

Comment:
@bfabio @sebbalex while this works, it's not very cool.

Could you pls find a nicer way to do it?